extensions/dashlane-vault/src/lib/dcli.ts

```diff
@@ -2,21 +2,34 @@ import { getPreferenceValues } from "@raycast/api";
 import { existsSync } from "fs";
 import { safeParse } from "valibot";
 
-import { execFilePromis } from "@/helper/exec";
+import { getMasterPassword } from "@/helper/master-password";
 import { VaultCredential, VaultCredentialSchema, VaultNote, VaultNoteSchema } from "@/types/dcli";
+import { exec } from "child_process";
```

Comment:
Why to you add execa and then still use exec?

Reply:
execa did not work for some reason when executing that particular command it threw some weird error.